avm/res/storage/storage-account/blob-service/main.bicep

```diff
@@ -28,8 +28,8 @@ param containerDeleteRetentionPolicyDays int?
 @description('Optional. This property when set to true allows deletion of the soft deleted blob versions and snapshots. This property cannot be used with blob restore policy. This property only applies to blob service and does not apply to containers or file share.')
 param containerDeleteRetentionPolicyAllowPermanentDelete bool = false
 
-@description('Optional. Specifies CORS rules for the Blob service. You can include up to five CorsRule elements in the request. If no CorsRule elements are included in the request body, all CORS rules will be deleted, and CORS will be disabled for the Blob service.')
-param corsRules array = []
+@description('Optional. The List of CORS rules. You can include up to five CorsRule elements in the request.')
```

Comment:
Actually I see the corsRuleType is the same for all children. What about having it defined and exported in the parent and imported in the children to avoid code duplication? Any side effects you can think of with this approach?

Reply:
I meant local import, not public one

Reply:
Which I addressed with the 2nd sentence. 😉

Reply:
I'll need to investigate a bit on this. As mentioned my goal would be to avoid defining the same type 4 times. However, I don't want to block the PR on it as functionality-wise this solution and my probably not feasible one would be the same. Hence approving.